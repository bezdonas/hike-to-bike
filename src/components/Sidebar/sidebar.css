.sidebar {
  position: relative;
  text-align: center;
  float: left;
  box-sizing: border-box;
  padding: 20px;
  width: 250px;
  height: 100vh;
  z-index: 100;
  box-shadow: 5px 0px 20px rgba(0, 0, 0, 0.5);
}
