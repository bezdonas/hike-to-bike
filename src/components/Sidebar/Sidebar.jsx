import { h, Component } from 'preact';
import './sidebar.css';

export default class Sidebar extends Component {
  render() {
    return <aside className="sidebar">{this.props.children}</aside>;
  }
}
