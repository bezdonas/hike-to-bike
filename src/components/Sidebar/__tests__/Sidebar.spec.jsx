import { h, preact } from 'preact';
import render from 'preact-render-to-json';
import Sidebar from '../Sidebar.jsx';

describe('Sidebar', () => {
  it('renders children and matches snapshot', () => {
    expect(render(<Sidebar>I am inside sidebar!</Sidebar>)).toMatchSnapshot();
  });
});
