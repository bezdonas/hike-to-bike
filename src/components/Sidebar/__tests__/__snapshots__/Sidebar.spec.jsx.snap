// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar renders children and matches snapshot 1`] = `
<aside
  class="sidebar"
>
  I am inside sidebar!
</aside>
`;
